src/ui/MLFacet/MLFacetValues/MLFacetValues.ts

```diff
@@ -86,13 +86,13 @@ export class MLFacetValues {
   }
 
   private buildShowLess() {
-    const showLess = $$('button', { className: 'coveo-ml-facet-show-less', ariaLabel: l('ShowLess') }, `- ${l('ShowLess')}`);
```

Comment:
Why is the ariaLabel no longer needed? Same comment for line 95.

Reply:
It's the same as the content now, I don't think it's needed anymore is it?